GHzDACs/ghz_fpga_server.py

```diff
@@ -1644,18 +1659,66 @@ def dac_run_sram(self, c, data, loop=False, blockDelay=0):
         parameters specifies the number of microseconds to delay
         for a multiblock sequence.
         """
-        if not len(data):
-            return
-        
-        if loop:
-            # make sure data is at least 20 words long by repeating it
-            data *= (20-1)/len(data) + 1
-        else:
-            # Set data at least 20 words long by repeating first value
-            data = list(data) + [data[0]] * (20-len(data))
-        
+        if len(data) < 20:
+            raise ValueError("Cannot play less than 20 ns of data.")
```

Comment:
1. It is a JT restriction, as 20 ns length means idling over cell 4, which means a from address of 2 (for the END op), which is not spaced far enough from the start address of 0.
2. The previous (pre-JT) behavior was to silently modify the sequence to 20 ns long. That felt wrong to me. Should we change it back?